app/components/agenda/agendaitem/agendaitem-case-panel/agendaitem-case-panel-view.hbs

```diff
@@ -104,8 +108,13 @@
         <Auk::Toolbar as |Toolbar|>
           <Toolbar.Group @position="left" as |Group|>
             <Group.Item>
-              <pre class="auk-u-text-pre-line" data-test-agendaitem-titles-comment>
-                <strong class="auk-u-text-bold">{{t "comment-title"}}: </strong>{{@agendaitem.comment}}
+              <pre
+                class="auk-u-text-pre-line"
+                data-test-agendaitem-titles-comment
+              >
+                {{! Strong opening and closing on 1 line to prevent whitespace to be rendered. }}
+                {{! prettier-ignore }}
+                <strong class="auk-u-text-bold">{{t "comment-title"}}:</strong>{{@agendaitem.comment}}
```

Comment:
with the oneliner, there is a space missing between `{{t "comment-title"}}:` and `</strong>`